mParticle-Button/MPKitButton.m

```diff
@@ -197,4 +228,22 @@ - (void)observeAttributionTokenDidChangeNotification:(NSNotification *)note {
     }
 }
 
+
+- (NSArray<ButtonProduct *> *)buttonProductsFromProducts:(NSArray <MPProduct *> *)products {
+    NSMutableArray *buttonProducts = [NSMutableArray array];
+    for (MPProduct *product in products) {
+        ButtonProduct *buttonProduct = [ButtonProduct new];
+        buttonProduct.name = product.name;
+        buttonProduct.id = product.sku;
+        buttonProduct.value = (NSInteger)(product.price.doubleValue * 100);
```

Comment:
Will this throw an exception if `product.price` is unavailable?

Reply:
Nope, will end up `0`